content/post/Android_Root101.en.md

```diff
@@ -87,11 +91,11 @@ _Reboot the Device._
 
 **_Congratulations! Now you have a phone with a CUSTOM ROM of your own choice with no bloatwares._**
 
-______
 ______
 
-*If during the process you encoutered any issues do look up your issues on XDA or else Google your issues, you'll surely find a solution.*
+*If during the process you encoutered any issues, do look up your issues on XDA or else Google your issues, you'll surely find a solution.*
 
-*If you enjoyed reading this article do let me know by tweeting to me on my __twitter-handle @RohanVTK__. Your input will highly appreciable as well.*
+*If you found this article helpful, do let me know by tweeting to me on my __twitter-handle @RohanVTK__.*
```

Comment:
`[RohanVTK](https://twitter.com/RohanVTK)`

Keeping some of that a11y in mind.